hapi/build.gradle.kts

```diff
@@ -34,7 +34,7 @@ tasks.cloneHederaProtobufs {
     // uncomment below to use a specific tag
     // tag = "v0.53.0"
     // uncomment below to use a specific branch
-    branch = "main"
+    branch = "add-throttle-at-consensus"
```

Comment:
intended?

Reply:
yes, because services develop will be broken if we just merge protobufs first. Once we point protobufs changes in services, this should be fine to not change these branches.